build/requirements.txt

```diff
@@ -30,6 +31,8 @@ nltk==3.9.1
 nmslib==2.1.1
 plotly==5.24.1
 plotnine==0.13.5
+openai-clip
+torchvision
```

Comment:
add a specific version to prevent future breaks due to compatibility. Note: we have deprecation warnings coming from the open information extraction notebook right now, presumably because a newer version of pytorch is being imported than is used by one of the libraries that chapter uses. Possibly once the torchvision version (and underlying pytorch version) need to be lowered?

Reply:
I've locked openai-clip to 1.0.1 and torchvision to 0.20.1 as the codebase is currently functioning on.=

Reply:
I've locked openai-clip to 1.0.1 and torchvision to 0.20.1 as the codebase is currently functioning with